build/configs/rtl8721csm/tc/defconfig

```diff
@@ -1222,6 +1222,7 @@ CONFIG_EXAMPLES_TESTCASE_KERNEL=y
 # CONFIG_DEBUG_TC_KN is not set
 CONFIG_TC_KN_ALL=y
 CONFIG_TC_KERNEL_CLOCK=y
+CONFIG_TC_KERNEL_SYSTEM_DEBUG=y
```

Comment:
I think you enable this manually. Because CONFIG_DEBUG_SYSTEM is not enabled even it has a dependency with that.

Reply:
corrected. please check now.